consensus_encoding/Cargo.toml

```diff
@@ -20,6 +20,9 @@ alloc = ["internals/alloc"]
 [dependencies]
 internals = { package = "bitcoin-internals", path = "../internals", version = "0.4.0" }
 
+[dev-dependencies]
+hex = { package = "hex-conservative", version = "0.3.0" }
```

Comment:
Should we use version 1.0.0 of hex?

Reply:
Unless I'm mistaken, 1.0.0 doesn't have the BytesToHexIter. Only the 0.x.0 versions do.

Reply:
Ah, my bad. Just found these docs (in case helpful for others) on the [stabilization strategy for hex](https://github.com/rust-bitcoin/hex-conservative?tab=readme-ov-file#stabilization-strategy). So not the craziest idea to target types in the unstable version at the moment.